src/main/resources/org/openstreetmap/atlas/geography/atlas/items/complex/islands/islands.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Any reason this one is in a different package than the others?

Reply:
I tried to keep islands and water separate, hence different packages.